The exception message for ``inspect.getfile()`` now correctly reports the passed class rather than the builtins module.